hack/machineset.sh

```diff
@@ -78,7 +78,7 @@ get_aws_ms() {
   local provider=$4
 
   # get the AMI id for the Windows VM
-  ami_date="2020.10.14"
+  ami_date="2021.01.13"
```

Comment:
We are going to have to constantly update this. We wanted to hard code this for predictability but the fact that AWS will constantly change this, I suggest making this script smarter i.e. grab the latest `Windows_Server-2019-English-Full-ContainersLatest` image.

Reply:
Forgot to mention that this can be done in a follow up PR.

Reply:
Ok, should I create a ticket for that and then create a follow up PR? And how should I proceed with this current PR?

Reply:
>Ok, should I create a ticket for that and then create a follow up PR? 

Correct

>And how should I proceed with this current PR?

We can merge after you fix the doc section as mentioned in my previous comment.